rdmo/projects/progress.py

```diff
@@ -49,6 +49,13 @@ def compute_sets(values):
 
 def compute_next_relevant_page(current_page, direction, catalog, resolved_conditions):
     # compute the next relevant page based on resolved conditions.
+
+    # Determine the next page based on the specified direction
+    next_page = (
+        catalog.get_prev_page(current_page) if direction == 'prev'
+        else catalog.get_next_page(current_page)
+    )
+
     # If no current_page, return None
     if not current_page:
```

Comment:
some `current_page` need to be renamed to `next_page`

Reply:
snap, thanks! Ive updated it now.

Reply:
yes, now also with comments